app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -26,6 +31,25 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['$http', '$scope',
     miqService.sparkleOff();
   }
 
+  $scope.storageManagerChanged = function (id) {
+    miqService.sparkleOn();
+    
+    $http.get('/cloud_volume/storage_manager_form_fields/' + id)
```

Comment:
A single call is preferred since that would mean a single trip to the server (preferred) as opposed to multiple requests (not preferred).

Going forward, we want all our GET and POST requests to be actual API calls, which means no new routes have to be added or no new UI controller code to support that route.